docs/source/testing/larva/larva.rst

```diff
@@ -20,4 +20,5 @@ Here is the table of contents:
 
    stubbing
    services
+   options
```

Comment:
I saw that your pull request is in branch "master" on your personal GitHub account. This will give you trouble when I work on the manual while your work is being reviewed. I propose that next time you use another branch. Then you can synchronize with my work on the Frank!Manual.